packages/base/package-scripts.js

```diff
@@ -6,12 +6,16 @@ const port = `9191`;
 
 const assetParametersScript = resolve.sync("@ui5/webcomponents-base/lib/generate-asset-parameters/index.js");
 
+const generateHash = resolve.sync("@ui5/webcomponents-tools/lib/hash/generate.js");
+const hashIsUpToDate = resolve.sync("@ui5/webcomponents-tools/lib/hash/upToDate.js");
+const UP_TO_DATE = `node ${hashIsUpToDate} dist/ hash.txt && echo "Up to date."`;
+
 const scripts = {
-	clean: "rimraf dist",
+	clean: "rimraf dist && rimraf hash.txt",
```

Comment:
why not keep the hash.txt file? it will mismatch dist anyway and will force a new build, and will not surprise developers by showing up in their git as removed.